extension.js

```diff
@@ -56,7 +56,8 @@ function getMouseHoveredWindowActor() {
 function onScroll(actor, event) {
   Log.debug("on scroll");
   win_actor = getMouseHoveredWindowActor();
-  let opacity = win_actor.get_opacity();
+  let win_surface = win_actor.get_children();
```

Comment:
Thanks for the commit. I think this would fix the no-wallpaper-background bug?(I saw this: https://gitlab.gnome.org/GNOME/mutter/-/tree/master/src/compositor) 
MetaSurfaceActor seems to be a newly-added concept of mutter. In order to keep the backward-compatibility, i will later on add some version-check code.

Reply:
Yeah, right, now it's just a hotfix for 3.3 with no-wallpaper-background. If i will have a time i will try to help you with backward-compatibility.